Dockerfile

```diff
@@ -1,12 +1,19 @@
-FROM  node 
-#comment
-
+FROM node:latest
 MAINTAINER mikegcoleman@gmail.com
 
-# Bundle app source
-COPY . /src
```

Comment:
This isn't optimized to utilize the Docker cache properly; below updated lines allow for optimization and not adding unnecessary files  (e.g. - `.git` directory, Dockerfile, etc)